src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/UpdateInterpreterDetailsConfirmation.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.handlers.postsubmit;
+
+import static java.util.Objects.requireNonNull;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.Event;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.Callback;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.PostSubmitCallbackResponse;
+import uk.gov.hmcts.reform.iacaseapi.domain.handlers.PostSubmitCallbackHandler;
+
+@Component
+public class UpdateInterpreterDetailsConfirmation implements PostSubmitCallbackHandler<AsylumCase> {
+
+    public boolean canHandle(
```

Comment:
Is this formatting normal? Why not have the parameters on the same line as the method name? Same for the next ones.

Reply:
Also for lots of the statements in the methods, you spread them over multiple lines but is there a reason to do that?

Reply:
Yes you're right that they don't look normal, I should have refactored them after copying from other classes, thanks.